msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -54,11 +54,21 @@ Create the conversation, after you have the user or channel information.
 
 ## Create the conversation
 
-Create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+You can create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+
+To create the conversation, you need a `userId`, `tenantId`, and `serviceUrl`.
+
+For `serviceUrl`, use the value from an incoming activity triggering the flow or one of the global service URLs. If the `serviceUrl` isn't available from an incoming activity triggering the proactive scenario, use the following global URL endpoints:
+
+* Public: `https://smba.trafficmanager.net/teams/`
+* GCC: `https://smba.infra.gcc.teams.microsoft.com/gcc`
+* GCCH: `https://smba.infra.gov.teams.microsoft.us/gcch`
```

Comment:
Shall we use teams as cloud alias for GCC and GCCH? Also how do we expect the developers to know a request is from gov cloud?

Reply:
had an offline discussion with Ryan and doc team. The teams alias is not tested in gov clouds yet so we will keep it as is.

Docs teams created an ADO item to track the update once test is ready:
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/2509953

Here's the parent ADO:
https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/2383655